### PR TITLE
filesystem: Remove duplication.

### PR DESCRIPTION
**Pull request by jodh-intel**
Remove duplication in filesystem.createAllResources().

Signed-off-by: James O. D. Hunt <james.o.hunt@intel.com>

### CODE REVIEW

File: filesystem.go [hunk 1]
```diff
@@ -103,44 +103,37 @@ type resourceStorage interface {
...
+			_, path, _ := fs.containerURI(pod.id, container.id, resource)
+			err = os.MkdirAll(path, os.ModeDir)
+			if err != nil {
+				fs.deletePodResources(pod.id, nil)
+				return err
+			}
 		}
 	}
 
-	podlockFile, _, err := fs.podURI(pod.id, lockFileType)
```
Comment (sboeuf) [inline filesystem.go]: What is the reason why you prefer this syntax rather than the previous one ?

File: filesystem.go [hunk 2]
```diff
@@ -103,44 +103,37 @@ type resourceStorage interface {
...
+	var podlockFile string
+	podlockFile, _, err = fs.podURI(pod.id, lockFileType)
 	if err != nil {
 		fs.deletePodResources(pod.id, nil)
 		return err
 	}
 
 	_, err = os.Stat(podlockFile)
 	if err != nil {
-		lockFile, err := os.Create(podlockFile)
```
Comment (sboeuf) [inline filesystem.go]: Ditto.

File: filesystem.go (hunk 1 above)
Comment (jodh-intel) [inline filesystem.go]: ah - I thought a new `err` would get created, but seemingly not. Updated.

**Comment by sboeuf** [comment]
lgtm

**Comment by sameo** [comment]
LGTM

**Pull request merged**